lib/client/api_service.es6.js

```diff
@@ -23,33 +23,48 @@ angular.module('confluence').service('api', ['$window', function($window) {
   const BROWSER_SPECIFIC_URL = ORIGIN + '/browser-specific';
   const AGGRESSIVE_REMOVAL_URL = ORIGIN + '/aggressive-removal';
 
+  const ctx = foam.box.Context.create();
+  // Names must match those registered in in server box context.
+  function getClientDAO(name, cls) {
+    return foam.dao.StreamingClientDAO.create({
+      of: cls,
+      delegate: foam.box.SubBox.create({
+        name: name,
+        delegate: foam.box.WebSocketBox.create({
```

Comment:
Hopefully this helps: https://github.com/foam-framework/foam2/blob/f66420803fc22bba315b64b7d24345ed17001893/src/foam/box/README.md